src/tpm2_pytss/utils.py

```diff
@@ -71,6 +71,47 @@ def make_credential(
     return (credblob, secret)
 
 
+def credential_to_tools(
+    id_object: TPM2B_ID_OBJECT, encrypted_secret: TPM2B_ENCRYPTED_SECRET
```

Comment:
also support raw bytes here and then in the code check if the input is an instanceof bytes and call the proper TPM2B constructor, ie something like `TPM2B_ENCRYPTED_SECRET(bytes)`.

Reply:
Done!